app/loader/KafkaManagerLoader.scala

```diff
@@ -39,6 +40,8 @@ class ApplicationComponents(context: Context) extends BuiltInComponentsFromConte
   private[this] lazy val assetsC = new controllers.Assets(httpErrorHandler)
   private[this] lazy val webJarsAssetsC = new controllers.WebJarAssets(httpErrorHandler, context.initialConfiguration, context.environment)
 
+  override lazy val httpFilters = Seq(BasicAuthenticationFilter(context.initialConfiguration))
```

Comment:
Would be great if you can add configuration to enable it.  Not everyone may want to enable it.